ftp.go

```diff
@@ -701,6 +701,21 @@ func (c *ServerConn) RemoveDir(path string) error {
 	return err
 }
 
+// Quotas issues a SITE QUOTA command.
+// This will read all params from the list and convert them to Uppercase Camel Case for ease of use.
+func (c *ServerConn) Quotas() (entries map[string]string, err error) {
+	_, message, err := c.cmd(-1, "SITE QUOTA")
+	if err != nil {
+		return
+	}
+
+	if message != "" {
+		entries, _ := parseQuotas(message)
+		return entries, nil
+	}
+	return nil, err
```

Comment:
`err` is always nil?

Reply:
I've made an update to this, Essentially if no errors are thrown from `c.cmd` then yes it's `nil`.